src/gc/impl/conservative/gc.d

```diff
@@ -383,9 +396,49 @@ struct GC
             return res;
     }
 
-    /**
-     *
-     */
+
+    void enable()
```

Comment:
It would be easier to review if you could avoid moving functions. If you think it's necessary it would be nicer in a separate commit.